libraries/AP_DDS/README.md

```diff
@@ -111,7 +111,7 @@ Run the simulator with the following command. If using UDP, the only parameter y
 | SERIAL1_PROTOCOL | Set this to 45 to use DDS on the serial port | 0 |
 ```console
 # Wipe params till you see "AP: ArduPilot Ready"
-# Select your favorite vehicle type
+# Select your favourite vehicle type
```

Comment:
was not misspelled

Reply:
> was not misspelled

Well... that's a matter of some debate :-)  I think the key thing is that we don't correct one way or the other and accept either.